posthog/test/ai/openai/test_openai.py

```diff
@@ -83,6 +83,7 @@ def mock_openai_response_with_responses_api():
         usage=ResponseUsage(
             input_tokens=10,
             output_tokens=10,
+            input_tokens_details={"prompt_tokens": 10, "cached_tokens": 0},
```

Comment:
was this a missing type or something?  Feels not in scope with what we're doing lol

Reply:
Yeah, mypy kept complaining so I fixed it.